CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Added client for read topics
```

Comment:
Extended the `ydb.Connection` interface with experimental `db.Topic()` client (control plane and reader API)